ext/LuxReactantExt/layers.jl

```diff
@@ -0,0 +1,30 @@
+# Embedding
+function (e::Lux.Embedding)(x::TracedRNumber{<:Reactant.ReactantInt}, ps, st::NamedTuple)
+    return ps.weight[:, x], st
+end
+
+# Recurrent Layers
+function (r::Lux.Recurrence{False})(x::AnyTracedRArray, ps, st::NamedTuple)
+    if r.ordering isa Lux.TimeLastIndex ||
+       (r.ordering isa Lux.BatchLastIndex && ndims(x) == 2)
+        idxs = ntuple(Returns(Colon()), ndims(x) - 1)
+        (out, carry), st = r.cell(x[idxs..., 1], ps, st)
+        T = size(x, ndims(x))
+        @trace for i in 2:T
```

Comment:
@Pangoraw this is how I am doing the iterations. It is statically inferable from the size of the array. Should I rewrite this in some way?